CHANGELOG.md

```diff
@@ -64,6 +65,9 @@ Drop `--main-only` flag from all `neofs-cli object head` commands.
 Drop IR's `timers.stop_estimation.*` and `timers.distribute_basic_income` configuration values, they are not used anymore.
 `neofs-adm fschain estimations` was removed.
 
+Use new `fschain.disable_autodeploy` IR configuration option instead of deprecated `fschain_autodeploy`,
+that was removed in next release.
```

Comment:
mention that default value for the old config has changed?

Reply:
but in fact, i think it is useless work, we will just drop it in the next release, nobody gonna change this exact value in their configs, they will just use the new one